exercises/practice/allergies/test/allergies_test.dart

```diff
@@ -1,275 +1,259 @@
-        final bool result = allergies.allergicTo('pollen', 255);
-        expect(result, equals(true));
-      }, skip: true);
-    });
-
-    group('testing for cats allergy', () {
-      test('not allergic to anything', () {
-        final bool result = allergies.allergicTo('cats', 0);
-        expect(result, equals(false));
-      }, skip: true);
-
-      test('allergic only to cats', () {
-        final bool result = allergies.allergicTo('cats', 128);
-        expect(result, equals(true));
-      }, skip: true);
-
-      test('allergic to cats and something else', () {
-        final bool result = allergies.allergicTo('cats', 192);
-        expect(result, equals(true));
-      }, skip: true);
-
-      test('allergic to something, but not cats', () {
-        final bool result = allergies.allergicTo('cats', 64);
-        expect(result, equals(false));
-      }, skip: true);
-
-      test('allergic to everything', () {
-        final bool result = allergies.allergicTo('cats', 255);
-        expect(result, equals(true));
-      }, skip: true);
-    });
-
-    group('list when:', () {
-      test('no allergies', () {
-        final List<String> result = allergies.list(0);
-        expect(result, equals(<String>[]));
-      }, skip: true);
-
-      test('just eggs', () {
-        final List<String> result = allergies.list(1);
-        expect(result, equals(<String>['eggs']));
-      }, skip: true);
-
-      test('just peanuts', () {
-        final List<String> result = allergies.list(2);
-        expect(result, equals(<String>['peanuts']));
-      }, skip: true);
-
-      test('just strawberries', () {
-        final List<String> result = allergies.list(8);
-        expect(result, equals(<String>['strawberries']));
-      }, skip: true);
-
-      test('eggs and peanuts', () {
-        final List<String> result = allergies.list(3);
-        expect(result, equals(<String>['eggs', 'peanuts']));
-      }, skip: true);
-
-      test('more than eggs but not peanuts', () {
-        final List<String> result = allergies.list(5);
-        expect(result, equals(<String>['eggs', 'shellfish']));
-      }, skip: true);
-
-      test('lots of stuff', () {
-        final List<String> result = allergies.list(248);
-        expect(result, equals(<String>['strawberries', 'tomatoes', 'chocolate', 'pollen', 'cats']));
-      }, skip: true);
-
-      test('everything', () {
-        final List<String> result = allergies.list(255);
-        expect(
-            result,
-            equals(
-                <String>['eggs', 'peanuts', 'shellfish', 'strawberries', 'tomatoes', 'chocolate', 'pollen', 'cats']));
-      }, skip: true);
-
-      test('no allergen score parts', () {
-        final List<String> result = allergies.list(509);
-        expect(
-            result, equals(<String>['eggs', 'shellfish', 'strawberries', 'tomatoes', 'chocolate', 'pollen', 'cats']));
-      }, skip: true);
-    });
+    test('testing for eggs allergy - not allergic to anything', () {
```

Comment:
I do like it the other way when there are groups of tests that could easily fall in the same batches.

But I just don't know if toml would be able to denote that.

Reply:
I'll implement grouping and regenerate the test suite, and we can go from there 👍